BAZProjectC1/Models/Strings.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Strings.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+enum EndpointsList:String  {
```

Comment:
Ten cuidado con el espaciado
```suggestion
enum EndpointsList: String  {
```